gcb/rbe/configs/path/cc/BUILD

```diff
@@ -38,7 +38,10 @@ filegroup(
 
 filegroup(
     name = "compiler_deps",
-    srcs = glob(["extra_tools/**"], allow_empty = True) + [":builtin_include_directory_paths"],
+    srcs = glob(
```

Comment:
Ah, I thought these were new config changes. It is just formatting chagnes from BUILDIFIER?

(This file was autogenerated from rbe helper scripts. )

Reply:
Yes, this is just the automatic application of `buildifier`'s fixes.